### ISSUE TITLE
Error writing to DB when running 2.0.0 version

### ISSUE THREAD

**Issue by SteveFreeman** [WASdev/tool.lars#183]
I am getting this error in the log:
```
com.mongodb.WriteConcernException: Write failed with error code 79 and error message 'No write concern mode named 'JOURNALED' found in replica set configuration'
```
What changes do I need to make to the MongoDB replica set?

**Comment by BillyD73** [comment]
Hi @SteveFreeman - could you please tell me which versions of the MongoDB client and server you are using as well as your Java and Gradle versions, and the steps to reproduce the problem please? I have been running on Mongodb client and server v4.2.0 using Java 8 and haven't been able to reproduce this issue.

**Comment by SteveFreeman** [comment]
MongoDB 4.0.12 and IBM Java 8.0.5.36.  Gradle has nothing to do with the runtime as this is happening after the build is complete and when I deploy to a real environment.

**Comment by BillyD73** [comment]
Hi @SteveFreeman. Thank you for your patience whilst I have investigated this issue. I've pushed a fix to my `dev_bd` branch which should solve this problem. Please let me know how it works for you.

**Comment by BillyD73** [comment]
This has been fixed with the [2.0.1 release](https://github.com/WASdev/tool.lars/releases/tag/v2.0.1)
